tests/test_matrix_utils.py

```
@@ -5230,7 +5230,13 @@ async def test_login_matrix_bot_reuses_existing_device_id(
     mock_main_client.close = AsyncMock()
 
     with (
-        patch("mmrelay.matrix_utils.get_base_dir", return_value="/tmp"),
+        patch(
+            "mmrelay.paths.resolve_all_paths",
+            return_value={
+                "credentials_path": "/tmp/credentials.json",
```

Ruff (0.14.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/credentials.json"

(S108)